templates/bundler_audit.rake

```diff
@@ -4,9 +4,26 @@ if Rails.env.development? || Rails.env.test?
   namespace :bundler do
     desc "Updates the ruby-advisory-db and runs audit"
     task :audit do
+      fix_bundler_audit_issue_114_to_exit_on_update_failure
       %w(update check).each do |command|
         Bundler::Audit::CLI.start [command]
       end
```

Comment:
Thanks for the feedback! 

It could be: `Bundler::Audit::CLI.start ["check"]`. Where's a good spot to put the call to `update`?